src/applications/personalization/dashboard/helpers.jsx

```diff
@@ -82,7 +82,7 @@ export const sipEnabledForms = new Set([
   '22-5495',
   '40-10007',
   'VIC',
-  'complaint-tool'
+  'COMPLAINT-TOOL'
```

Comment:
@erikphansen previously recommended refactoring this so there aren't lots of magical strings. Issue created for this work [here](#12576).

Reply:
broken link to the issue?

Reply:
Thanks Erik, fixed.